modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/BulkActions.js

```diff
@@ -165,7 +165,9 @@ function BulkActions({
 										selectedItemsValue.length
 									} ${Liferay.Language.get(
 										'of'
-									)} ${total} Liferay.Language.get('items-selected')`}
+									)} ${total} ${Liferay.Language.get(
```

Comment:
Hey @oliv-yu, when taking a look on https://liferay.design/lexicon/core-components/toolbars/management-bar/, looks like it should be lowercase.

Also, why this language key isn't: `{0}-of-{1}-items-selected`? Then, you can use `sub` function there like `Liferay.Util.sub(Liferay.Language.get({0}-of-{1}-items-selected), selectedItemsValue.length, total)`